### PR TITLE
[Documentation] Docker version update [SKIP CI]

### LINKED ISSUE #2035: docker daemon failed to start after installing vDVS plugin
This issue was reported on the slack channel for docker-volume-vsphere. Posting the details from the information posted by the customer on the slack channel.

Issue as reported by customer:

"Just tried hatchway for the first time.  After running the following command the docker daemon bombed out and will no longer restart even after rebooting the vm:"
```
docker plugin install --grant-all-permissions --alias vsphere vmware/docker-volume-vsphere:0.19`
```

Journalctl logs:
```Error starting daemon: couldn't create plugin manager: failed to restore plugins: error reading /var/lib/docker/plugins/15fdae39e1
97a5348936b980a825af7795bb7494b72fe9e4df6dec3d458a328c/config.json: open /var/lib/docker/plugins/15fdae39e197a5348936b980a825af7795bb7494b72fe9e4df6dec3d458a328c/config.json
: no such file or directory
Dec 10 22:38:06 docker-client systemd[1]: docker.service: Main process exited, code=exited, status=1/FAILURE
Dec 10 22:38:06 docker-client systemd[1]: Failed to start Docker Daemon.
Dec 10 22:38:06 docker-client systemd[1]: docker.service: Unit entered failed state.
Dec 10 22:38:06 docker-client systemd[1]: docker.service: Failed with result 'exit-code'.```

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
fixes #2035

For the issue #2035, we have decided to update our document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +31,16 @@ Please check [VDVS Installation User Guide](http://vmware.github.io/vsphere-stor
 
 ## Supported Platforms
 
-**ESXi:** 6.0 and above<br />
-**Docker (Linux):** 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)<br />
-**Docker (Windows):** 1.13/17.03 and above (Windows containers mode only)
+**ESXi:** 6.0U2 and above<br />
+**Docker (Linux):** 17.06.1 and above to use managed plugin (v17.06.1 is having an important fix for data loss issue)<br />
```
Comment (tusharnt) [inline README.md]: Please remove these details - v17.06.1 is having an important fix for data loss issue

File: README.md [hunk 2]
```diff
@@ -31,9 +31,16 @@ Please check [VDVS Installation User Guide](http://vmware.github.io/vsphere-stor
...
+**Docker (Windows):** 17.06 and above (Windows containers mode only)<br />
+**Guest Operating System**:
+- Ubuntu 14.04 or higher (64 bit)
+   - Needs Upstart or systemctl to start and stop the service
+   - Needs [open vm tools or VMware Tools installed](https://kb.vmware.com/selfservice/microsites/search.do?language=en_US&cmd=displayKC&externalId=340) ```sudo apt-get install open-vm-tools```
+- RedHat and CentOS
```
Comment (tusharnt) [inline README.md]: Please split RedHat and CentOS on different lines.
Please mention the specific RHEL & CentOS versions that are validated.

File: README.md (hunk 1 above)
Comment (shuklanirdesh82) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (shuklanirdesh82) [reply README.md]: removed `CentOS`  part and added specific version for RHEL

**Pull request merged**